tests/Integration/Behaviour/Features/Scenario/CustomerService/customer_service_summary.feature

```diff
@@ -0,0 +1,17 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s customer_service --tags customer-management
+@restore-all-tables-before-feature
+@customer-management
+Feature: Customer service summary
+
+  Scenario: Add customer thread
+    When I create a contact "CONTACT-1" with following properties:
+      | localisedTitles        | Support |
```

Comment:
preferable (we use "z" everywhere else)
```suggestion
      | localizedTitles        | Support |
```

Reply:
nop, atleast everywhere its localisedTitles. If i change it then it will be the only place where its called like this

Reply:
It might be that customer tests were done this wrong way from beginning. Ok then don't mind it. (just keep in mind for next time then)